app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    f_name = command.split(" ")[1]
```

Comment:
Consider using more descriptive variable names instead of `f_name` and `d_file`. For example, `source_file` and `destination_file` would be clearer.